default-plugins/status-bar/src/tip/data/mod.rs

```diff
@@ -88,5 +89,25 @@ lazy_static! {
                 full: compact_layout::compact_layout_full,
             }
         ),
+        (
+            "move_tabs",
+            TipBody {
+                short: move_tabs::move_tabs_short,
+                medium: move_tabs::move_tabs_medium,
+                full: move_tabs::move_tabs_full,
+            }
+        )
     ]);
 }
+
+#[macro_export]
+macro_rules! ansi_strings {
+    ($ANSIStrings:expr) => {{
+        let strings: &[::ansi_term::ANSIString] = $ANSIStrings;
+        let ansi_strings = ::ansi_term::ANSIStrings(strings);
+        LinePart {
+            part: format!("{}", ansi_strings),
+            len: ::ansi_term::unstyled_len(&ansi_strings),
+        }
+    }};
+}
```

Comment:
It's a good refactoring, but could you leave this out of this PR and make it a separate PR?

(In my opinion, separating PR according to purpose makes management more convenient.)

Reply:
Yeah, I was thinking about that as well. I'll move the changes to separate PR :+1:

Reply:
Moved [here](https://github.com/zellij-org/zellij/pull/3086)